docs/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: quarkus-pinot
+title: Quarkus - Pinot
```

Comment:
```suggestion
title: Quarkus Pinot
```

This will organize it alphabetically

Reply:
This is a generated file, maybe this should be updated on the Quarkiverse template side

Reply:
Yeah, I know, it's on my TODO list for a while, it's time to send a PR :)

Reply:
@gastaldi I wanted to have a look at some of the easy fixes for Quarkiverse generation this afternoon if the release goes well. I will let you know.